docker-compose.yml

```diff
@@ -12,6 +11,12 @@ services:
       - "8080:8080"
     environment:
       - PYTHONUNBUFFERED=1
+      - DB_ENGINE=${DB_ENGINE}
+      - DB_NAME=${DB_NAME}
+      - DB_USER=${DB_USER}
+      - DB_PASSWORD=${DB_PASSWORD}
+      - DB_HOST=${DB_HOST}
+      - DB_PORT=${DB_PORT}
```

Comment:
Ensure that the environment variable `DB_PORT` is defined in your environment or `.env` file. Otherwise, the application may fail to start due to missing configuration.